en/console-and-shells.rst

```diff
@@ -354,6 +354,19 @@ By default, a ``requested`` extra param is automatically added when a Shell
 is dispatched using ``dispatchShell()``. This ``requested`` parameter prevents
 the CakePHP console welcome message from being displayed on dispatched shells.
 
+Exit Status Codes
+-----------------
+It can sometimes be useful to return exit status codes. Simply use php's ``exit()`` statement.
```

Comment:
maybe $this->error() can stay like it is and not return implicitly (in future).
return $this->exit(1, $optionalMsg) would call $this->error($msg) and could internally call exit() if not called in a testing context. But see the alternative suggestion below as an interim solution so that users don't get confused?